src/shogun/mathematics/linalg/LinalgBackendBase.h

```diff
@@ -228,7 +228,20 @@ class LinalgBackendBase
 	DEFINE_FOR_ALL_PTYPE(BACKEND_GENERIC_SET_CONST, SGVector)
 	DEFINE_FOR_ALL_PTYPE(BACKEND_GENERIC_SET_CONST, SGMatrix)
 	#undef BACKEND_GENERIC_SET_CONST
-	
+
+	/**
+	 * Wrapper method that sets const values to vectors or matrices.
```

Comment:
@karlnapf Would you suggest to change it to `set_col_const` or just remove it? I didn't see any call on this method in Shogun